pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py

```diff
@@ -241,6 +298,10 @@ def test_ptp_cmd_convert_negative(self):
         ptp_3 = Ptp(goal=object(), vel_scale=EXP_VEL_SCALE, acc_scale=EXP_ACC_SCALE)
         self.assertRaises(NotImplementedError, ptp_3._cmd_to_request, self.robot)
 
+        # 4
+        ptp_4 = Ptp(goal="123456", vel_scale=EXP_VEL_SCALE, acc_scale=EXP_ACC_SCALE)
```

Comment:
```suggestion
        ptp_4 = Ptp(goal="some_string", vel_scale=EXP_VEL_SCALE, acc_scale=EXP_ACC_SCALE)
```

Reply:
clarified the intention in using this string

Reply:
So even with the clarified explanation in the tests description you think it would be better to test for "some_string"?

Reply:
@agutenkunst Why?

Reply:
Imho it would state more clearly that this could be any string not just one containing numbers. But ok.